x-pack/plugins/apm/public/context/environments/use_enviroments.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { useContext } from 'react';
+import { EnvironmentContext } from './environment_context';
+
+export function useEnvironments() {
```

Comment:
Almost all of the other hooks in the context directory (except `useUrlParams` and this) are `useSomethingContext`. I believe that's a naming convention we established recently. Is it something we want to continue following with this?

Reply:
This was a thing I introduced recently to distinguish between different type of hooks. 
Before, `useEnvironment` (or something like that) would do an actual fetch in every component it was used, whereas `useTransactionType` would only subscribe to context thus only fetching data once regardless of the number of consumers.

Reply:
hungarian notation, I like it! (agree that it's more clear this way)